.github/workflows/check_precompiled.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Pass updated files in $updated_files environment variable
```

Comment:
_Conventionally_ environment variables are always created as `$UPPERCASE_VARIABLES`, but for such a simple one-off CI script I guess this doesn't really matter.